rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/TaskResultImpl.java

```diff
@@ -57,6 +58,8 @@
 
     private Map<String, byte[]> propagatedVariables;
 
+    private Map<String, Serializable> JobMap = new ConcurrentHashMap<>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "JobMap" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)